README.md

```
@@ -1,2 +1,77 @@
+- Google OAuth
+- Framer Motion
+- TailwindCSS
+- Shadcn UI
+- React Query
+- React Hook Form
+- Zod
+
+## Kurulum
+
+1. Repoyu klonlayın:
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)